packages/cli-kit/src/public/node/api/admin.ts

```diff
@@ -19,8 +25,9 @@ import {TypedDocumentNode} from '@graphql-typed-document-node/core'
 export async function adminRequest<T>(query: string, session: AdminSession, variables?: GraphQLVariables): Promise<T> {
   const api = 'Admin'
   const version = await fetchLatestSupportedApiVersion(session)
-  const url = adminUrl(session.storeFqdn, version)
-  return graphqlRequest({query, api, url, token: session.token, variables})
+  const url = adminUrl(session.storeFqdn, version, session)
+  const addedHeaders: {[header: string]: string} = headers(session)
```

Comment:
The return for `headers` is already defined so we don't have to redeclare it here.
```suggestion
  const addedHeaders = headers(session)
```